srcalc/src/main/java/gov/va/med/srcalc/web/view/PopulatedDisplayGroup.java

```diff
@@ -73,19 +74,21 @@ public PopulatedDisplayGroup(final List<? extends Variable> variables, final Pat
             fDisplayItems.add(visitor.getView(var, ""));
         }
         
-        // These group names are all well-known and are hard-coded as such.
-        if(group.getName().equalsIgnoreCase(CLINICAL_GROUP))
-        {
-            final List<String> factorList = new ArrayList<String>();
-            // Output the date without the time of day.
-            final DateTimeFormatter format = DateTimeFormat.forPattern("MM/dd/yy");
-            for(final HealthFactor factor: patient.getHealthFactors())
-            {
-                factorList.add(String.format("%s %s%n", format.print(factor.getDate()), factor.getName()));
-            }
-            final DisplayItem refInfo = new ReferenceItem("Health Factors", variables.get(0).getGroup(), factorList);
-            fDisplayItems.add(0, refInfo);
-        }
+        addReferenceInfo(group, patient);
+    }
+    
+    /**
+     * Constructs an instance and populates the groups with any patient notes that are
+     * automatically retrieved. This constructor should only be used to place reference information
+     * into the group after all PopulatedDisplayGroups have been built for variables.
+     * @param group the VariableGroup to which the reference information belongs
+     * @param patient the current patient in the calculation
+     */
+    public PopulatedDisplayGroup(final VariableGroup group, final Patient patient)
+    {
+        fVariables = ImmutableList.of();
```

Comment:
Hmm, now this makes `equals()` problematic: a Medications group and a Clinical Conditions group with no variables will be counted equal. Defining it properly will be hard, though: do we really need `equals()`?